cmd/validatorcmd/getBalance.go

```diff
@@ -58,68 +71,137 @@ func getBalance(_ *cobra.Command, _ []string) error {
 		return err
 	}
 
-	var balance uint64
-	if validationIDStr != "" {
-		validationID, err := ids.FromString(validationIDStr)
-		if err != nil {
-			return err
-		}
-		balance, err = txutils.GetValidatorPChainBalanceValidationID(network, validationID)
-		if err != nil {
-			return err
-		}
-		ux.Logger.PrintToUser("  Validator Balance: %.5f", float64(balance)/float64(units.Avax))
+	validationID, cancel, err := getNodeValidationID(network, l1, nodeIDStr, validationIDStr)
+	if err != nil {
+		return err
+	}
+	if cancel {
 		return nil
 	}
+	if validationID == ids.Empty {
+		return fmt.Errorf("the specified node is not a L1 validator")
+	}
 
-	isBootstrapValidator, err := app.Prompt.CaptureYesNo("Is the validator a bootstrap validator?")
+	balance, err := txutils.GetValidatorPChainBalanceValidationID(network, validationID)
 	if err != nil {
 		return err
 	}
-	if isBootstrapValidator {
-		if l1 == "" {
-			return fmt.Errorf("--l1 flag is required to get bootstrap validator balance")
-		}
-		sc, err := app.LoadSidecar(l1)
+	ux.Logger.PrintToUser("  Validator Balance: %.5f AVAX", float64(balance)/float64(units.Avax))
+
+	return nil
+}
+
+func getNodeValidationID(
+	network models.Network,
+	l1 string,
+	nodeIDStr string,
+	validationIDStr string,
+) (ids.ID, bool, error) {
```

Comment:
nit: 
it's not not clear what `bool` in return means.
pls change func name or add comment for this